isobus/src/isobus_virtual_terminal_working_set_base.cpp

```diff
@@ -112,44 +112,15 @@ namespace isobus
 								// Next, parse macro list
 								if (iopLength >= sizeOfMacros)
 								{
```

Comment:
I'd say we can also remove this check and just rely on the helper function instead.

```cpp
retVal = parse_object_macro_reference(tempObject, numberOfMacrosToFollow, iopData, iopLength);
if (!retVal)
{
    LOG_ERROR("[WS]: Failed to parse macro references for object " + isobus::to_string(static_cast<int>(decodedID)));
}
```